hedera-node/src/main/resources/throttles-perf.json

```diff
@@ -0,0 +1,73 @@
+{
+    "buckets": [
+        {
+            "name": "ThroughputLimits",
+            "burstPeriod": 1,
+            "throttleGroups": [
+                {
+                    "opsPerSec": 12000,
+                    "operations": [
+                        "CryptoCreate", "CryptoTransfer", "CryptoUpdate", "CryptoDelete", "CryptoGetInfo", "CryptoGetAccountRecords",
+                        "ConsensusCreateTopic", "ConsensusSubmitMessage", "ConsensusUpdateTopic", "ConsensusDeleteTopic", "ConsensusGetTopicInfo",
+                        "TokenGetInfo",
+                        "ScheduleDelete", "ScheduleGetInfo",
+                        "FileGetContents", "FileGetInfo",
+                        "ContractUpdate", "ContractDelete", "ContractGetInfo", "ContractGetBytecode", "ContractGetRecords", "ContractCallLocal", 
+                        "TransactionGetRecord",
+                        "GetVersionInfo"
+                    ]
+                },
+                {
+                    "opsPerSec": 13,
+                    "operations": [ "ContractCall", "ContractCreate", "FileCreate", "FileUpdate", "FileAppend", "FileDelete" ]
+                },
+                {
+                    "opsPerSec": 3000,
+                    "operations": [
+                        "ScheduleSign", 
+                        "TokenCreate", "TokenDelete", "TokenMint", "TokenBurn", "TokenUpdate", "TokenAssociateToAccount", "TokenAccountWipe",
+                        "TokenDissociateFromAccount","TokenFreezeAccount", "TokenUnfreezeAccount", "TokenGrantKycToAccount", "TokenRevokeKycFromAccount"
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "PriorityReservations",
+            "burstPeriod": 1,
+            "throttleGroups": [
+                {
+                    "opsPerSec": 10,
+                    "operations": [ "ContractCall", "ContractCreate", "FileCreate", "FileUpdate", "FileAppend", "FileDelete" ]
+                }
+            ]
+        },
+        {
+            "name": "CreationLimits",
+            "burstPeriod": 10,
+            "throttleGroups": [
+                {
+                    "opsPerSec": 2,
+                    "operations": [ "CryptoCreate" ]
+                },
+                {
+                    "opsPerSec": 5,
+                    "operations": [ "ConsensusCreateTopic" ]
+                },
+                {
+                    "opsPerSec": 100,
+                    "operations": [ "TokenCreate", "TokenAssociateToAccount", "ScheduleCreate" ]
+                }
```

Comment:
Should these have higher numbers as the name says `perf` ?

Reply:
These throttles just match what Tom Z. always ask me to update perfnet to. :)